flink-core/src/main/java/org/apache/flink/api/common/restartstrategy/RestartStrategies.java

```diff
@@ -162,6 +162,14 @@ public boolean equals(Object obj) {
 			}
 		}
 
+		@Override
+		public String toString() {
+			return "FixedDelayRestartStrategyConfiguration{" +
```

Comment:
Why is this being added? (and why not also for the other configurations?)

Reply:
You are right. I should've added it to all classes. I added it for my debugging. I added a common implementation to the base class that reuses the description.